library/bigip_software.py

```diff
@@ -46,64 +40,148 @@
     choices:
       - yes
       - no
-  hotfix:
-    description:
-      - The path to an optional Hotfix to install. This parameter requires that
-        the C(software) parameter be specified.
-    required: false
-    default: None
-    aliases:
-      - hotfix_image
   reuse_inactive_volume:
     description:
-      - Automatically chooses the first inactive volume in alphanumeric order. If there
-        is no inactive volume, new volume with incremented volume name will be created.
-        For example, if HD1.1 is currently active and no other volume exists, then the
-        module will create HD1.2 and install the software. If volume name does not end with
-        numeric character, then add .1 to the current active volume name. When C(volume)
-        is specified, this option will be ignored.
+      - Automatically chooses the first inactive volume in alphanumeric order. 
+        If there is no inactive volume, new volume with incremented volume name 
+        will be created. For example, if HD1.1 is currently active and no other 
+        volume exists, then the module will create HD1.2 and install the 
+        software. If volume name does not end with numeric character, 
+        then add .1 to the current active volume name. When C(volume) is 
+        specified, this option will be ignored.
```

Comment:
make sure you provide an example for this in the EXAMPLES variable to illustrate this